### PR TITLE
Jeanie

### PR DESCRIPTION
**Pull request by jeaniewu**
This is mostly updating the walls (get to seaparate them) so a lot of changes to the scenes

Feature:
- Falling down the Ditch
- Input is now following the Input manager
- ghostWall anim is there (update can be done thru the prefab now)

### CODE REVIEW

**Comment by jeaniewu** [comment]
@jayfriso @Gowtham100 code review please :)

**Comment by Gowtham100** [comment]
Looks good to me :+1:

File: Assets/GhostWall.cs [hunk 1]
```diff
@@ -0,0 +1,15 @@
+﻿using UnityEngine;
+using System.Collections;
+
+public class GhostWall : MonoBehaviour {
```
Comment (jayfriso) [inline Assets/GhostWall.cs:4]: What is this file for? It doesn't have anything.

File: Assets/Scripts/Ditch.cs [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+	}
+
+	void OnTriggerEnter2D(Collider2D other) {
+		if (other.gameObject.CompareTag ("Player")) {
+			player = other.gameObject;
+			StartCoroutine ("Fall");
+		}
+	}
+
+	IEnumerator Fall() {
```
Comment (jayfriso) [inline Assets/Scripts/Ditch.cs:26]: [JUST A COMMENT] Is this how you run while loops across multiple update frames? That's pretty cool!

File: Assets/Scripts/receiveSignal/receiveSignalGhostWall.cs [hunk 3]
```diff
@@ -0,0 +1,42 @@
...
+		}
+	}
+
+	public override void deactivate(){
+		foreach (GhostWall wall in ghostWalls) {
+			deactivateChild (wall.GetComponent<Collider2D> (), wall.GetComponent<Renderer> ());
+		}
+	}
+
+	public void activateChild(Collider2D collider2D2, Renderer renderer2){
```
Comment (jayfriso) [inline Assets/Scripts/receiveSignal/receiveSignalGhostWall.cs:23]: Maybe these methods (activateChild, and deactivateChild) should be in another script for each child wall. As the method is more related to the individual wall  component and not the whole wall. Then this script can just call the method in each wall component.

File: Assets/Scripts/Ditch.cs (hunk 2 above)
Comment (jeaniewu) [inline Assets/Scripts/Ditch.cs:26]: really cool right!!

**Comment by jeaniewu** [comment]
@jayfriso most of the ghostwall code is temporary as we will be changing it later, so you can neglect it for now, i will clean it up

**Comment by jayfriso** [comment]
I gave it a :+1: but from now on you should try to keep features as seperate as possible (or atleast do them as sepereate commits). Anyway good job though!

**Comment by jeaniewu** [comment]
@jayfriso yeaa will keep it as separate commits

**Pull request merged**